crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -1,9 +1,6 @@
 use bytes::Bytes;
 use ibc_types::{
-    core::{
-        channel::ChannelId,
-        client::Height as IbcHeight,
-    },
+    core::{channel::ChannelId, client::Height as IbcHeight},
```

Comment:
general throughout here, please don't override formatting. 

If you have just installed can run `just fmt rust` to format according to repo standards.